### DISCUSSION TITLE
Use multiple AWS accounts

### DISCUSSION THREAD

**Discussion by flemzord** [aws/copilot-cli#2868] [Q&A] [answered]
Hello, 

I would like to use several AWS accounts to separate my environments (staging / production). 
However, I can't see how I could do that. 

Is this supported in copilot? 

Regards, 
Maxence

**Comment by efekarakus** [comment, marked as answer]
Hi @flemzord 👋 

This is supported with Copilot today!

When running `copilot env init` you can choose which AWS profile to use when creating a "staging" or "production" environment.  The region associated with the profile will be used to create your environment (VPC, LB, ...) and services.

For example:
![Screen Shot 2021-09-23 at 1 28 34 PM](https://user-images.githubusercontent.com/879348/134579214-52216e13-8120-4ec2-bf76-8482c644023e.png)



(docs: https://aws.github.io/copilot-cli/docs/credentials/#environment-credentials)
